### PR TITLE
MES-5620 update cat BE, C, D office pages for Delegated Examiners fails

### PR DESCRIPTION
**Pull request by Cullima**
## Description
- updates office page for failed tests

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by tomcrawleyy — APPROVED** [review]

File: src/pages/office/cat-d/office.cat-d.page.ts [hunk 1]
```diff
@@ -459,6 +461,14 @@ export class OfficeCatDPage extends BasePageComponent {
         select(getTestData),
         map(data => this.faultCountProvider.getDrivingFaultSumCount(this.testCategory as TestCategory, data)),
       ),
+      seriousFaultCount$: currentTest$.pipe(
+        select(getTestData),
+        map(data => this.faultCountProvider.getSeriousFaultSumCount(TestCategory.BE, data)),
```
Comment (andrewsetterfield) [inline src/pages/office/cat-d/office.cat-d.page.ts]: Cat BE?

File: src/pages/office/cat-d/office.cat-d.page.ts [hunk 2]
```diff
@@ -459,6 +461,14 @@ export class OfficeCatDPage extends BasePageComponent {
...
+      ),
+      dangerousFaultCount$: currentTest$.pipe(
+        select(getTestData),
+        map(data => this.faultCountProvider.getDangerousFaultSumCount(TestCategory.BE, data)),
```
Comment (andrewsetterfield) [inline src/pages/office/cat-d/office.cat-d.page.ts]: Cat BE?

**Review by andrewsetterfield — APPROVED** [review]

File: src/pages/office/cat-d/office.cat-d.page.ts (hunk 1 above)
Comment (Cullima) [reply src/pages/office/cat-d/office.cat-d.page.ts]: thanks for spotting. amended

File: src/pages/office/cat-d/office.cat-d.page.ts (hunk 2 above)
Comment (Cullima) [reply src/pages/office/cat-d/office.cat-d.page.ts]: thanks for spotting. amended

**Pull request merged**